hi-ml-azure/src/health/azure/himl_tensorboard.py

```diff
@@ -142,15 +150,16 @@ def main() -> None:  # pragma: no cover
     )
     parser.add_argument(
         "--run_recovery_ids",
-        default=None,
-        action='append',
+        default=[],
+        nargs="+",
         required=False,
         help="Optional run recovery ids of the runs to plot"
     )
     parser.add_argument(
         "--run_ids",
         default=[],
         nargs="+",
+        required=False,
         help="Optional run ids of the runs to plot"
     )
```

Comment:
Should we merge those args, and accept any run ID, run_recovery ID, or combination thereof? Could also shorten the arg name to "--runs"

Reply:
Personally I prefer the idea of keeping them explicit but we can discuss